iidm/iidm-serde/src/test/resources/xsd/terminalMockNoSerDe.xsd

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema version="1.13"
+<xs:schema version="1.14"
            xmlns:xs="http://www.w3.org/2001/XMLSchema"
-           xmlns:iidm="http://www.powsybl.org/schema/iidm/1_13"
-           targetNamespace="http://www.powsybl.org/schema/iidm/ext/terminal_mock_no_serde/1_12"
+           xmlns:iidm="http://www.powsybl.org/schema/iidm/1_14"
+           targetNamespace="http://www.powsybl.org/schema/iidm/ext/terminal_mock_no_serde/1_0"
```

Comment:
Here should it not be 1_14?

Reply:
This xsd is not used by a "versionable" SerDe + it is only used in unit tests. Therefore, we don't need to change it's version at each IIDM version.
This will save us changes in `notFoundExtension.xml` and `skippedExtensions.xml` (where it is used).